src/hooks/web3-constants.js

```diff
@@ -0,0 +1,142 @@
+import _CONTRACTS from 'https://contracts.webaverse.com/config/addresses.js';
```

Comment:
Why is this file under `hooks`?

Imo constants should be exposed higher up. We already have a constants file, and although this one is web3 specific, I would never look for it here in a React subdirectory.

Reply:
It's dead constant.
I removed it.